controllers/storageclaim_controller.go

```diff
@@ -651,6 +678,69 @@ func (r *StorageClaimReconciler) hasVolumeSnapshotContents() (bool, error) {
 	return false, nil
 }
 
+func (r *StorageClaimReconciler) getCephFSVolumeGroupSnapshotClass(data map[string]string, labels map[string]string) *groupsnapapi.VolumeGroupSnapshotClass {
+	volumgroupesnapshotclass := &groupsnapapi.VolumeGroupSnapshotClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   r.storageClaim.Name,
+			Labels: labels,
+		},
+		Driver:         csi.GetCephFSDriverName(),
+		DeletionPolicy: snapapi.VolumeSnapshotContentDelete,
+		Parameters:     data,
+	}
+	return volumgroupesnapshotclass
+}
+
+func (r *StorageClaimReconciler) getCephRBDVolumeGroupSnapshotClass(data map[string]string, labels map[string]string) *groupsnapapi.VolumeGroupSnapshotClass {
+	volumegroupsnapshotclass := &groupsnapapi.VolumeGroupSnapshotClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   r.storageClaim.Name,
+			Labels: labels,
+		},
+		Driver:         csi.GetRBDDriverName(),
+		DeletionPolicy: snapapi.VolumeSnapshotContentDelete,
+		Parameters:     data,
+	}
+	return volumegroupsnapshotclass
+}
```

Comment:
can we rework on this and existing code for separate RBD and cephfs by just passing the driver name so that we can get rid of duplicate functions?